release-notes/2.1/2.1.12/2.1.801-sdk-download.md

```diff
@@ -0,0 +1,155 @@
+[aspnetcore-runtime-win-x86.exe]: https://download.visualstudio.microsoft.com/download/pr/d8b36b96-890f-4207-9418-565fa6e7b64f/775b6912b82886fdb7be25b63a47bc57/aspnetcore-runtime-2.1.12-win-x86.exe
+[aspnetcore-runtime-win-x86.zip]: https://download.visualstudio.microsoft.com/download/pr/dbd566a6-3629-45bf-804f-79e86d6c6a93/ed8a595e8db40013264f8f304423ed6a/aspnetcore-runtime-2.1.12-win-x86.zip
+[dotnet-hosting-win.exe]: https://download.visualstudio.microsoft.com/download/pr/eebd54bc-c3a2-4580-bb29-b35c1c5ffa92/22ffe5649861167d3d5728d3cb4b10a1/dotnet-hosting-2.1.12-win.exe
+
+[//]: # ( SDK 2.1.801 )
+[dotnet-sdk-linux-arm.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/08b5ee2d-85d3-4e9a-8392-a53d9bfc45b6/8e2bc9a61074f8a82a455a0614226262/dotnet-sdk-2.1.801-linux-arm.tar.gz
+[dotnet-sdk-linux-arm64.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/2612b592-2f37-4a80-9630-5ebca0420f87/fabe268b88545aaf10b24fdeb66f745c/dotnet-sdk-2.1.801-linux-arm64.tar.gz
+[dotnet-sdk-linux-musl-x64.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/583f70d4-af97-42e9-bea8-ea66dc9ace5a/be02e62f143925cc8d3c24696d29a937/dotnet-sdk-2.1.801-linux-musl-x64.tar.gz
+[dotnet-sdk-linux-x64.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/b95d51d0-6be0-49eb-aae7-4092c77634d3/4dd3723dcee0d50ad5e5ead5f4f4127e/dotnet-sdk-2.1.801-linux-x64.tar.gz
+[dotnet-sdk-osx-x64.pkg]: https://download.visualstudio.microsoft.com/download/pr/faa8e3e4-285d-4bcc-b6b1-2545e1b52ee3/91ba59bd2d30a2fc20b0b08bbc4673f7/dotnet-sdk-2.1.801-osx-x64.pkg
+[dotnet-sdk-osx-x64.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/f2b22dbe-6802-4051-a3a2-fc0defd772f0/3e1ac7d36dbe27d38c9a8e5a415bb2ae/dotnet-sdk-2.1.801-osx-x64.tar.gz
+[dotnet-sdk-rhel.6-x64.tar.gz]: https://download.visualstudio.microsoft.com/download/pr/e48d6be6-541f-415e-91d2-6a53a47b8b7e/c12ccaf8c9622e5fe8b129b2fa80a994/dotnet-sdk-2.1.801-rhel.6-x64.tar.gz
+[dotnet-sdk-win-x64.exe]: https://download.visualstudio.microsoft.com/download/pr/29f92590-ac92-45f0-99e8-e60c767dc4e9/ddc1014a788613364b5308d6c49db3db/dotnet-sdk-2.1.801-win-x64.exe
+[dotnet-sdk-win-x64.zip]: https://download.visualstudio.microsoft.com/download/pr/a3e45ac9-c70d-4526-b05d-1fe621b146e1/55cce526bf94100d95db61fcf65cc182/dotnet-sdk-2.1.801-win-x64.zip
+[dotnet-sdk-win-x86.exe]: https://download.visualstudio.microsoft.com/download/pr/408b9eb6-c213-4498-abf3-317b73e2eb54/0ca48259be33b961af8980cd2bbaac51/dotnet-sdk-2.1.801-win-x86.exe
+[dotnet-sdk-win-x86.zip]: https://download.visualstudio.microsoft.com/download/pr/bf58f321-34f3-4fb4-a201-5922dacf6a20/67b37675235c144c8f4be49ea739a1d4/dotnet-sdk-2.1.801-win-x86.zip
+
+[checksums-runtime]: https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.12-runtime-sha.txt
+[checksums-sdk]: https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.701-sdk-sha.txt
```

Comment:
this still points to 2.1.701 instead of 2.1.801 checksums

Reply:
Thanks for reporting it. It is fixed now.